docs/src/pages/components/pickers/pickers.md

```diff
@@ -20,24 +20,53 @@ packageName: '@material-ui/lab'
 
 {{"demo": "pages/components/pickers/MaterialUIPickers.js"}}
 
+### Setup
+
+You need to provide a date-library that is used by the pickers by setting the `dateAdapter` to an adapter of your choosing.
+
+We currently support 4 different date-libraries:
+
+- [date-fns](https://date-fns.org/)
+- [Day.js](https://day.js.org/)
+- [Luxon](https://moment.github.io/luxon/#/)
+- [Moment.js](https://momentjs.com/)
+
+First you have to install the adapter package for the date-library you want to use:
+
+```sh
+// date-fns
+npm install @date-io/date-fns
+// or for Day.js
+npm install -s @date-io/dayjs
+// or for Luxon
+npm install -s @date-io/luxon
+// or for Moment.js
+npm install @date-io/moment
+```
+
+Then you have to set the `dateAdapter` prop of the `LocalizationProvider` accordingly:
+
+```js
+// date-fns
+import DateAdapter from '@material-ui/lab/AdapterDateFns';
+// or for Day.js
+import DateAdapter from '@material-ui/lab/AdapterDayjs';
+// or for Luxon
+import DateAdapter from '@material-ui/lab/AdapterLuxon';
+// or for Moment.js
+import DateAdapter from '@material-ui/lab/AdapterMoment';
+
+function App({ children }) {
+  return (
+    <LocalizationProvider dateAdapter={DateAdapter}>{children}</LocalizationProvider>
+  );
+}
+```
+
 ## Native pickers
 
 ⚠️ Native input controls support by browsers [isn't perfect](https://caniuse.com/#feat=input-datetime).
```

Comment:
Should we precede this caveat with a reason for using them? (Otherwise, why support them?)

Reply:
Good question. I honestly don't know. a11y is probably the most important factor here and I don't know how well our and the native pickers behave right now. Other than that it comes down to UI/UX I guess and I wouldn't recommend either one unless we can point to user studies.

Reply:
> consistent UX for the end-users

On mobile perhaps I think it's fair to say that native date/tome controls are almost never used on desktop.

A note covering those points would be a good addition.

Reply:
> A note covering those points would be a good addition.

I would defer this to when the pickers are out of the lab since we're all mostly grasping here. Once we have a clear picture about the pickers we can revisit.